src/images/trading_app/contracts/asian.svg

```diff
@@ -0,0 +1 @@
+<svg width="42" height="24" viewBox="0 0 42 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><title>asian_up_down_on</title><defs><circle id="a" cx="12" cy="12" r="12"/><circle id="b" cx="12" cy="12" r="12"/></defs><g fill="none" fill-rule="evenodd"><g transform="translate(18)"><use fill="#2A3052" xlink:href="#a"/><circle stroke="#E4E6F1" cx="12" cy="12" r="11.5"/></g><path d="M22 20h16V4H22z"/><path fill="#FFF" d="M23 14.667h13.333v-1.333H23zM23 10.667h13.333v-1H23zM23 8h13.333V7H23z"/><path fill="#FFF" fill-rule="nonzero" d="M29.765 10.093l5.541 6.427-1.01.87-4.728-5.483-1.348 1.011-4.737-5.83 1.034-.842 3.93 4.836z"/><path fill="#FFF" d="M35.576 17.6v-2.208l-1.006-1.007v2.19h-2.266l1.052 1.025z"/><g><g><use fill="#2A3052" xlink:href="#b"/><circle stroke="#E4E6F1" cx="12" cy="12" r="11.5"/></g><path d="M4 4h16v16H4z"/><path fill="#FFF" d="M5 9.333h13.333v1.333H5zM5 13.333h13.333v1H5zM5 16h13.333v1H5z"/><path fill="#FFF" fill-rule="nonzero" d="M11.765 13.907l5.541-6.427-1.01-.87-4.728 5.483-1.348-1.011-4.737 5.83 1.034.842 3.93-4.836z"/><path fill="#FFF" d="M17.576 6.4v2.208L16.57 9.615v-2.19h-2.266L15.356 6.4z"/></g></g></svg>
```

Comment:
do we need `xmlns:xlink="http://www.w3.org/1999/xlink"` attribute ?

Reply:
@kellybinary , I optimized the `svg` with `svgo`, didn't realize it kept this attribute in.. :D